target/linux/ramips/dts/mt7620a_dlink_dir-806a-b1.dts

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "dlink,dir-806a-b1", "ralink,mt7620a-soc";
+	model = "D-Link DIR-806A (rev. B1)";
```

Comment:
To match the format of other ramips D-Link models, shouldn't this be `D-Link DIR-806A B1`?

Reply:
> To match the format of other ramips D-Link models, shouldn't this be `D-Link DIR-806A B1`?

Ok. I fixed it.